internal/weaver/weavelet.go

```diff
@@ -54,10 +53,11 @@ var readyMethodKey = call.MakeMethodKey("", "ready")
 
 // A weavelet runs and manages components. As the name suggests, a weavelet is
 // analogous to a kubelet.
-type weavelet struct {
-	ctx       context.Context
-	env       env                  // Manages interactions with execution environment
-	info      *protos.EnvelopeInfo // Setup info sent by the deployer.
+type Weavelet struct {
+	ctx context.Context
+	env env // Manages interactions with execution environment
```

Comment:
TODO to delete env?

Reply:
Yup, I already deleted it in a PR I'm about to send out in a second.